test/cypress/integration/getting_started_spec.ts

```diff
@@ -332,6 +306,27 @@ describe('Getting started modal', () => {
         });
     });
 
+    it('only enables one environment to be selected at a time', () => {
+        goToNextStep();
+        cy.contains('button', 'AWS').click();
+        cy.contains('button.active', 'AWS');
+
+        cy.contains('button', 'GCP').click();
+        cy.contains('button.active', 'AWS').should('not.exist');
+        cy.contains('button.active', 'GCP');
+    });
+
+    it('enables to click "next" button when an environment is being selected', () => {
+        goToNextStep();
+        cy.contains('button', 'Next').should('to.be.disabled');
+
+        cy.contains('button', 'AWS').click();
+        cy.contains('button', 'Next').should('not.to.be.disabled');
+
+        cy.contains('button', 'AWS').click();
+        cy.contains('button', 'Next').should('to.be.disabled');
+    });
+
     it('should keep button and field states when navigating beetwen steps', () => {
```

Comment:
It's not your change, but maybe worth to fix at once:
```suggestion
    it('should keep button and field states when navigating between steps', () => {
```

Reply:
Sure, that change wouldn't take the whole sprint, so why not 😁

Reply:
Done